api/src/client/dynamics/license-status/DynamicsLicenseApplicationIdClient.ts

```diff
@@ -41,7 +41,7 @@ export const DynamicsLicenseApplicationIdClient = (
               applicationDetails.statecode === ACTIVE_STATECODE
           )
           .filter((applicationDetails: LicenseApplicationIdApiResponse) => {
-            if (licenseType === "Health Club") return true;
+            if (licenseType === "Health Club" || licenseType === "Telemarketing") return true;
```

Comment:
[sand] Given new information from DCA (from 2 days ago), we know that Public Movers is the only license type where `MAIN_APP_END_DIGITS` is used to identify a "main" application. Could we restructure this check to say "if public movers check last 2 digits, else return true"?

Reply:
yes sir...